java/src/jmri/jmrit/audio/JoalAudioSource.java

```diff
@@ -524,35 +532,32 @@ protected void doFadeOut() {
         if (log.isDebugEnabled()) {
             log.debug("Fade-out JoalAudioSource (" + this.getSystemName() + ")");
         }
-        if (_initialised && (isBound() || isQueued())) {
+        if (_initialised && audioAlive() && (isBound() || isQueued())) {
             AudioSourceFadeThread asft = new AudioSourceFadeThread(this);
             asft.start();
         }
     }
 
     @Override
     protected void cleanup() {
-        if (log.isDebugEnabled()) {
-            log.debug("Cleanup JoalAudioSource (" + this.getSystemName() + ")");
-        }
-        if (_initialised && (isBound() || isQueued())) {
+        log.debug("Cleanup JoalAudioSource ({})", this.getSystemName());
+        if (_initialised && audioAlive() && (isBound() || isQueued())) {
             al.alSourceStop(_source[0]);
             al.alDeleteSources(1, _source, 0);
             this._source = null;
-            log.debug("...done cleanup");
+            log.info("...done cleanup");
```

Comment:
This should remain a debug level statement unless the prior debug statement becomes an info statement.